### PR TITLE
Fix most pressing RTL issues

### PR DESCRIPTION
**Pull request by swissspidy**
## Summary

See #3012

Issues resolved with this PR:

* [x] Editor carousel layout
* [x] Highlighted drag areas for cross-page dragging
* [x] CTA button form overlapping the button

## Checklist

- [x] My pull request is addressing an [open issue](https://github.com/ampproject/amp-wp/contributing/project-management.md#life-of-an-issue) (please create one otherwise).
- [x] My code is tested and passes existing [tests](https://github.com/ampproject/amp-wp/contributing/engineering.md#tests).
- [x] My code follows the [Engineering Guidelines](https://github.com/ampproject/amp-wp/contributing/engineering.md) (updates are often made to the guidelines, check it out periodically).

### CODE REVIEW

File: assets/src/stories-editor/components/editor-carousel/index.js [hunk 1]
```diff
@@ -57,9 +60,14 @@ const EditorCarousel = () => {
 			wrapper.current.style.display = 'none';
 		} else {
 			wrapper.current.style.display = '';
-			wrapper.current.style.transform = `translateX(calc(50% - ${ PAGE_BORDER }px - ${ ( STORY_PAGE_INNER_WIDTH + STORY_PAGE_MARGIN ) / 2 }px - ${ ( currentIndex ) * STORY_PAGE_MARGIN }px - ${ currentIndex * STORY_PAGE_INNER_WIDTH }px))`;
+
+			if ( isRTL ) {
+				wrapper.current.style.transform = `translateX(calc(-50% - ${ PAGE_BORDER }px + ${ ( STORY_PAGE_INNER_WIDTH + STORY_PAGE_MARGIN ) / 2 }px + ${ ( currentIndex ) * STORY_PAGE_MARGIN }px + ${ currentIndex * STORY_PAGE_INNER_WIDTH }px))`;
```
Comment (barklund) [inline assets/src/stories-editor/components/editor-carousel/index.js]: Is there any reason why we're off-loading this calculation to css? Why not calculate the pixels in JS and only `calc(50% + ${x}px)`? Then we can also un-duplicate the logic, by setting `const factor = isRTL ? -1 : 1;` somewhere.

File: assets/src/stories-editor/blocks/amp-story-cta/edit.css [hunk 2]
```diff
@@ -125,6 +125,7 @@ div[data-type="amp/amp-story-cta"] .amp-overlay {
 	position: absolute;
 	z-index: 10;
 	margin-top: -0.5em;
+	top: 100%;
```
Comment (barklund) [inline assets/src/stories-editor/blocks/amp-story-cta/edit.css]: This seems like a rather substantial change. How can that be inconsequential for the LTR view?

Comment (swissspidy) [reply assets/src/stories-editor/blocks/amp-story-cta/edit.css]: I'd have to ask the CSS gods 😄

File: assets/src/stories-editor/components/editor-carousel/index.js (hunk 1 above)
Comment (swissspidy) [reply assets/src/stories-editor/components/editor-carousel/index.js]: Sure that might also work, I'll give it a try. Might make it more readable 👍

Comment (swissspidy) [reply assets/src/stories-editor/components/editor-carousel/index.js]: Actually, not so sure 🤔

**Comment by barklund** [comment]
@googlebot I consent.

File: assets/src/stories-editor/components/editor-carousel/index.js (hunk 1 above)
Comment (barklund) [reply assets/src/stories-editor/components/editor-carousel/index.js]: I tried a simplification in 36d5a4a. Feel free to revert if you don't find it simpler 🤷‍♂.

**Pull request merged**